contracts/protocol/bases/FundsBase.sol

```diff
@@ -228,8 +229,8 @@ abstract contract FundsBase is Context {
         }
 
         if (payoff.protocol > 0) {
-            increaseAvailableFunds(PROTOCOL_ENTITY_ID, exchangeToken, payoff.protocol);
-            emit IBosonFundsLibEvents.ProtocolFeeCollected(_exchangeId, exchangeToken, payoff.protocol, sender);
+            increaseAvailableFunds(0, exchangeToken, payoff.protocol);
```

Comment:
```suggestion
            increaseAvailableFunds(PROTOCOL_ENTITY_ID, exchangeToken, payoff.protocol);
```
I looks like `PROTOCOL_ENTITY_ID` got lost in one of the merges.I think it should be reinstated.